docs/web-apps/automated-testing/cypress.md

```diff
@@ -37,7 +37,7 @@ Sauce Labs supports the following test configurations for Cypress:
   <tbody>
     <tr>
       <td rowspan='2'>13.13.3</td>
-      <td rowspan='2'>20</td>
+      <td rowspan='2'>20, Version Configurable</td>
```

Comment:
What do you think of adding a new column for this? 

| Cypress Version | Node.js Version | Supports configurable runtime |
| -----------------| --------------- | ------------------------------- |
| 13.13.3 | 20 | ✅ |

Reply:
👍 I was a bit hesitant about adding an extra column. Looks like good to have one.